main.js

```diff
@@ -0,0 +1,74 @@
+// grab the customer container
+const customerContainer = document.querySelector("#customers");
+
+// date conversion function
+function dateConvert(date) {
+  return moment(date).format("ll");
+}
+
+// loop through the customer objects and build html for each one
+for (let customer of customers) {
+  let card = document.createElement("div");
+  card.classList.add("card");
```

Comment:
These could be combined in one line. You *can* add multiple classes at once to a classList